### PR TITLE
Add use cases

### LINKED ISSUE #14: Update Developer Guide use cases

### PR DESCRIPTION
**Pull request by wes-w-z-h**
Add use cases for current features
Add use cases for incoming features

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -303,32 +306,188 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+
+- 2a. The list does not contain the concert.
+
+  Use case ends.
+
+- 4a. The list does not contain the person.
+
+  Use case ends.
+
+- 5a. The given person/ concert is invalid.
```
Comment (bbbbcccd) [inline docs/DeveloperGuide.md]: Space after / should be removed to indicate or

**Review by bbbbcccd — COMMENTED** [review]
lgtm with minor changes

**Comment by wes-w-z-h** [comment]
Guide on [how to use /](https://www.grammarly.com/blog/punctuation-capitalization/slash/)

**Review by bbbbcccd — APPROVED** [review]
lgtm!

**Pull request merged**